core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/Resource.scala

```diff
@@ -86,6 +87,24 @@ object Resource extends ConfigFormat {
     }
   }
 
+  final class DMGFile(val path: Path) extends IntellijResource {
+    def installTo(target: Path): Unit = {
+      import sys.process._
+
+      // attach disk image from .dmg file to local filesystem
+      val dmgDir = s"${System.getProperty("java.io.tmpdir")}dmg_dir"
+      s"mkdir $dmgDir".!
+      s"hdiutil attach -mountpoint $dmgDir ${path.toString}".!
```

Comment:
Out of curiosity, what's the lifetime of such mount... until the machine is rebooted? (`/tmp/` might be special in this respect as its cleaned every now and then on reboot anyway; details differ between different Unix-like OSes, however)

Reply:
I don't know that